src/datapilot/core/platforms/dbt/insights/checks/check_model_has_tests_by_name.py

```diff
@@ -0,0 +1,69 @@
+from typing import List
+
+from datapilot.config.utils import get_test_name_configuration
+from datapilot.core.insights.utils import get_severity
+from datapilot.core.platforms.dbt.insights.checks.base import ChecksInsight
+from datapilot.core.platforms.dbt.insights.schema import DBTInsightResult
+from datapilot.core.platforms.dbt.insights.schema import DBTModelInsightResponse
+from datapilot.core.platforms.dbt.schemas.manifest import AltimateResourceType
+
+
+class CheckModelHasTestsByName(ChecksInsight):
+    NAME = "Check Model Has Tests By Name"
```

Comment:
Not the same as checkpoint. They are checking something like 

```
not_null=2, unique=1
```

Reply:
Not sure what you mean. We are checking if the model has test by name